src/View.js

```diff
@@ -0,0 +1,257 @@
+                      return <FormattedMessage id="stripes-smart-components.searchResultsCountHeader" values={{ count }} />;
+                    }
+                    return <FormattedMessage id="stripes-smart-components.searchCriteria" />;
+                  }
+                  }
+                  paneTitle={<FormattedMessage id="ui-plugin-find-eresource.eresources" />}
+                >
+                  <MultiColumnList
+                    autosize
+                    columnMapping={{
+                      name: <FormattedMessage id="ui-plugin-find-eresource.prop.name" />,
+                      type: <FormattedMessage id="ui-plugin-find-eresource.prop.type" />,
+                      isbn: <FormattedMessage id="ui-plugin-find-eresource.prop.isbn" />,
+                      eissn: <FormattedMessage id="ui-plugin-find-eresource.prop.eissn" />,
+                      pissn: <FormattedMessage id="ui-plugin-find-eresource.prop.pissn" />,
+                    }}
+                    columnWidths={{
+                      name: 300,
+                      type: 100,
+                      isbn: 150,
+                      eissn: 150,
+                      pissn: 150,
+                    }}
+                    contentData={data.eresources}
+                    formatter={{
+                      name: e => e._object?.longName ?? e.name,
+                      type: e => <EResourceType resource={e} />,
+                      isbn: e => getResourceIdentifier(e._object, 'isbn'),
+                      eissn: e => getResourceIdentifier(e._object, 'eissn'),
+                      pissn: e => getResourceIdentifier(e._object, 'pissn') ?? getSiblingIdentifier(e._object, 'issn'),
+                    }}
+                    id="list-eresources"
+                    isEmptyMessage={
+                      source ? (
+                        <div data-test-eresources-no-results-message>
+                          <SearchAndSortNoResultsMessage
+                            filterPaneIsVisible
+                            searchTerm={query.query ?? ''}
+                            source={source}
+                            toggleFilterPane={toggleFilterPane}
+                          />
+                        </div>
+                      ) : '...'
+                    }
+                    isSelected={({ item }) => item.id === selectedRecordId}
+                    onHeaderClick={onSort}
+                    onNeedMoreData={onNeedMoreData}
+                    onRowClick={onSelectRow}
+                    pageAmount={100}
+                    pagingType="click"
+                    sortDirection={sortOrder.startsWith('-') ? 'descending' : 'ascending'}
+                    sortOrder={sortOrder.replace(/^-/, '').replace(/,.*/, '')}
+                    totalCount={count}
+                    virtualize
+                    visibleColumns={visibleColumns}
+                  />
+                </Pane>
+                {children}
```

Comment:
Do we ever even have `children` for the plugins? Seems like a carry-over from the main apps where they'll have child routes.